CONTRIBUTING.md

```diff
@@ -14,8 +14,9 @@ or just begin a dialog about the Persistent Memory Development Kit:
 * Chat with members of the PMDK team real-time on the **#pmem** IRC channel on [OFTC](http://www.oftc.net)
 
 **NOTE: If you do decide to implement code changes and contribute them,
-please make sure you agree your contribution can be made available
-under the [BSD-style License used for the Persistent Memory Development Kit](https://github.com/pmem/pmdk/blob/master/LICENSE).**
+please make sure you consent to the
+[BSD-style License used for the Persistent Memory Development Kit](https://github.com/pmem/pmdk/blob/master/LICENSE)
+under which your contribution will be made open-source.**
```

Comment:
I think this version is worse, so I have to ask: why do you want to change it?

Reply:
After offline discussion - this version, content-wise, is less correct. Reverting this change